### ISSUE TITLE
Not Found self.base_lrs And  self.scheduler.get_last_lr()

### ISSUE THREAD

**Issue by AndyGuo1** [JeffersonQin/YuzuMarker.FontDetection#27]
self.scheduler = CosineWarmupScheduler(
       optimizer, self.num_warmup_iters, self.num_iters
)
print("Current learning rate set to:", self.scheduler.get_last_lr())
        
class CosineWarmupScheduler(torch.optim.lr_scheduler._LRScheduler):

    def get_lr(self):
        lr_factor = self.get_lr_factor(epoch=self.last_epoch)
        return [base_lr * lr_factor for base_lr in self.base_lrs]

**Comment by JeffersonQin** [comment]
Please provide more context and detailed error information and stacktrace.

**Comment by AndyGuo1** [comment]
您好，发现在优化器这里self.base_lrs和self.scheduler.get_last_lr()没有找到定义，训练模型时，学习率初始值显示是0.

**Comment by JeffersonQin** [comment]
应该是pytorch lightning版本的问题，你回退到去年三月的版本试试

**Comment by JeffersonQin** [comment]
具体到底是哪个版本我已经不记得了，但是有个办法可以去查。

docker.io/jeffersonqin/yuzumarker.fontdetection.huggingfacespace.base:latest

你把上面这个docker镜像pull下来，然后exec bash进去做一下pip freeze就能知道当年的版本号了

我现在手边没电脑操作不了，你自己先研究一下吧，有问题随时follow up

**Comment by AndyGuo1** [comment]
嗯嗯好的，非常感谢
